.gitignore

```diff
@@ -27,5 +27,4 @@ node_modules
 # Users Environment Variables
 .lock-wscript
 
-docs/
```

Comment:
`docs` should be checked in and should have the docs you wrote, `doc` created when you generate the documentation for that module in isolation and those should not be checked in.